leakcanary-android-instrumentation/src/main/java/leakcanary/FailTestOnLeak.kt

```diff
@@ -0,0 +1,5 @@
+package leakcanary
+
+@Retention(AnnotationRetention.RUNTIME)
+@Target(AnnotationTarget.FUNCTION)
+annotation class FailTestOnLeak
```

Comment:
Can you add javadoc to explain how to use this and link to `FailAnnotatedTestOnLeakRunListener` ?

Reply:
Done!